app/src/main/java/hous/release/android/presentation/login/UserInputViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package hous.release.android.presentation.login
+
+import androidx.lifecycle.MediatorLiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import hous.release.android.util.extension.Event
+
+class UserInputViewModel : ViewModel() {
+    private val _nickname = MutableLiveData<String>()
+    val nickname get() = _nickname
+
+    private val _birthday = MutableLiveData<String>()
+    val birthday get() = _birthday
```

Comment:
캡슐화하는 이유가 해당 class 내에서만 수정할 수 있게 만들고, 해당 class 밖에서는 해당 변수를 수정하지 않고 읽게만 할 수 있도록 하는 것인데,

어처피 양방향 데이터 바인딩으로 인해서 xml 에서 해당 변수를 수정하니 굳이 캡슐화하지 않아도 될 것 같습니다

`요약`
```suggestion
    val nickname = MutableLiveData<String>()
    val birthday = MutableLiveData<String>()
```